en/docs/guides/login/client-side-app.md

```diff
@@ -73,9 +74,13 @@ Make the following requests via your application to connect your application to
 
 3. Validate the ID token. For the token request, you will receive a response containing the access token, scope, and ID token. The ID token contains basic user information. To check what is encoded within the ID token, you can use a tool such as <https://devtoolzone.com/decoder/jwt>.
 
+----
+
 !!! info "Related Topics"
-    - [Enable Login for a Sample OpenID Connect Web Application](../../../quick-starts/webapp-oidc-sample)
-    - [Manage User Sessions](insertlink)
```

Comment:
Is not this section done yet? I think we have completed it. Shall we check what are the existing docs and link them in this section.

Reply:
It has been linked below in line 83